source/components/GlobalNavigation/components/GlobalNavigationLink/GlobalNavigationLinkText.js

```diff
@@ -18,8 +18,12 @@ const GlobalNavigationLinkText = ({ link, isStandaloneLink }) => {
 };
 
 GlobalNavigationLinkText.propTypes = {
-    isStandaloneLink: PropTypes.bool.isRequired,
+    isStandaloneLink: PropTypes.bool,
     link: PropTypes.object.isRequired,
 };
 
+GlobalNavigationLinkText.defaultProps = {
+    isStandaloneLink: false,
```

Comment:
What's this change about? I don't see it used elsewhere in the PR.

Reply:
good catch - this one fixes prop types warning on the reference page. 

`isStandaloneLink` was supposed to be optional from the beginning, but I've marked is as `required` in the previous PR.